src/layers/generic.cpp

```diff
@@ -517,12 +543,12 @@ namespace marian {
 
     auto batchEmbeddings = apply(subBatch->data(), {dimWidth, dimBatch, dimEmb});
 #if 0
-    auto batchMask = graph->constant({dimWidth, dimBatch, 1},
-                                     inits::fromVector(subBatch->mask()));
-#else
     // experimental: hide inline-fix source tokens from cross attention
     auto batchMask = graph->constant({dimWidth, dimBatch, 1},
                                      inits::fromVector(subBatch->crossMaskWithInlineFixSourceSuppressed()));
+#else
+    auto batchMask = graph->constant({dimWidth, dimBatch, 1},
+                                     inits::fromVector(subBatch->mask()));
 #endif
```

Comment:
Here only the order of what is inside the #if part of the macro and in the #else part was switched. Mainly because this was trowing a couple of warnings if the tags <IDELIM> <IOPEN> and <ICLOSE> where not used.